nextflow.config

```diff
@@ -30,6 +30,7 @@ params {
     read_length                = 100
     starfusion_build           = true
     genomes                    = [:]
+    fusion_annot_lib           = null       // path to  dat.gz CTAT genome lib (e.g. https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/GRCh38_gencode_v44_CTAT_lib_Oct292023.plug-n-play.tar.gz)
```

Comment:
Should `https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/GRCh38_gencode_v44_CTAT_lib_Oct292023.plug-n-play.tar.gz` just become the default? Then it should still work out-of-the box @rannick 

The only difference is that now nextflow takes care of downloading the reference instead of the script calling `wget` internally.

Reply:
we want to update to gencode v46

Reply:
And do we want a 80GB file there?

Reply:
that's fair, but would it address your comment above? 

> as if will break the logic of self contained references in the pipeline, we need to provide an alternative

Reply:
I was also thinking about a plug-and-play option, but if we provide a s3 option, let's have the latest one there

Reply:
We can have a placeholder here until we have the latest s3 reference. For example, we could add the same original reference that was in the module (so no change in behaviour): `https://github.com/FusionAnnotator/CTAT_HumanFusionLib/releases/download/v0.3.0/fusion_lib.Mar2021.dat.gz`

and then open an issue to update it to the latest version when available in S3. Do you think that will make sense?

Reply:
Agree with the placeholder, it would address my comment about self-contained references. 
We can use the following link for the updated annotations it is the latest from CTAT as far as I know

https://sourceforge.net/projects/rnafusion/files/STAR-Fusion_gencodev44_CTAT_lib_Oct292925_resources_fusion_annot_lib.gz/download